public/scripts/app.js

```diff
@@ -2,6 +2,56 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+$.ajax({
+  method:"GET",
+  url: "/api/profile",
+  success: function (data) {
+  	$(".image-container").html(`<img src="${data.myFriends}" id="friends-image">`); 
+  }, 
+  error: handleError
+});
 
+$.ajax({
+  method:"GET",
+  url: "/api/cities",
+  success: handleSuccess,
+  error: handleError
 });
+
+$('form').on('submit', function(e) {
+	e.preventDefault();
+	$.ajax({
+		method: "POST",
+		url: "/api/cities",
+		data: $('form').serialize(),
+		success: function(data) {
+			renderCity(data);
+		},
+		error: handleError
+	})
+	$('input').val('');
```

Comment:
what does this input.val() do here ? you have already taken input data by form.serialize()